lib/translations.json

```diff
@@ -154,6 +154,18 @@
       "key": "withMembership/unauthorized/accountText",
       "value": "Konto ansehen"
     },
+    {
+      "key": "withMembership/ios/unauthorized/title",
+      "value": "Verlegerbereich"
+    },
+    {
+      "key": "withMembership/ios/unauthorized/explanation",
+      "value": "Sie haben keine aktive Mitgliedschaft. Besuchen Sie {webLink}, um mehr zu erfahren."
```

Comment:
Optional:
`<a>Erfahren Sie mehr</a>` instead of `Besuchen Sie <a>xxx</a>, um mehr zu erfahren`
(unless the intent is to actually promote the URL)